docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -389,6 +431,8 @@ const buildComponentDocumentation = async (options: {
           deprecated: !!deprecation || undefined,
           deprecationInfo:
             renderMarkdownInline(deprecation?.groups?.info || '').trim() || undefined,
+          signature,
+          additionalPropsInfo,
```

Comment:
```suggestion
          ...(Object.keys(additionalPropsInfo).length === 0 ? null : { additionalPropsInfo })
```

To remove empty objects `"additionalPropsInfo": {}`